main.go

```diff
@@ -110,6 +114,8 @@ func main() {
 	flag.BoolVar(&showVersion, "version", false, "outputs the version and exit")
 	flag.BoolVar(&enableScheduler, "enable-scheduler", false, "enable support for the NUMAResourcesScheduler object")
 	flag.BoolVar(&enableWebhooks, "enable-webhooks", false, "enable conversion webhooks")
+	flag.BoolVar(&enableWebhooks, "enable-metrics", false, "enable metrics server")
```

Comment:
you already have a flag (metricsAddr), wouldn't it be better to change the default value? Otherwise the cli ux is misleading, you set the address but nothing happens

Reply:
Yes, I'm still evaluating the best approach here